### ISSUE TITLE
Compile command suppresses compiler warnings

### ISSUE THREAD

**Issue by ilyar** [tonlabs/everdev#84]
```shell
node -v
v16.13.2
npm i -g everdev
npx everdev sol set --compiler latest --linker latest
wget --quiet https://raw.githubusercontent.com/ilyar/everscale-escrow/56b8ab4b8c2cb69a21300aef186472e06d8473dc/Escrow.sol
npx everdev sol compile Escrow.sol
# suppresses compiler warnings
```
expect
```shell
~/.everdev/solidity/solc Escrow.sol
Code was generated and saved to file Escrow.code
ABI was generated and saved to file Escrow.abi.json
Warning: Have no effect in TON. Delete this.
--> /home/ilyar/np/everdev/local/Escrow.sol:30:5:
|
30 |     payable
|     ^^^^^^^
```

**Comment by elasticLove1** [comment]
Thanks! Will check!)

**Comment by Artem-Zhdanov** [comment]
Hi, @ilyar! 
This issue has been fixed in the latest version of everdev. Can you test it?

**Comment by Artem-Zhdanov** [comment]
Fixed in 1.1.0, see https://github.com/tonlabs/everdev/blob/main/CHANGELOG.md#fixed-1
